object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -192,6 +192,7 @@ private void checkPseudoAssignToOwning(Node node) {
       AnnotationMirror cmAnno =
           typeFactory.getAnnotatedType(tree).getAnnotationInHierarchy(typeFactory.top);
       if (!calledMethodsSatisfyMustCall(mustCallVal, cmAnno)) {
+        checker.numMustCallFailed++;
```

Comment:
Rather than adding this before every call to `checker.reportError`, we could override `reportError` in `ObjectConstructionChecker` and add to this variable if the warning string is `"required.method.not.called"`.

Reply:
https://github.com/kelloggm/object-construction-checker/pull/249/commits/5260a7c49c6dc2c042e59816683cc968c422883c